content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,473 @@
+# How to build a task manager with angular.
+
+### Introduction
+As developers, it is common practice to build simple applications. This will help solidify one's knowledge about a particular programming language or framework. 
+
+In this article, we will build a simple task manager application (AKA To-do app) using [Angular](https://angular.io/docs). Angular is a JavaScript framework.
+
+### Table of content
+-  Prerequisites
+-  Introduction
+-  Key takeaways
+-  What is a Task Manager
+-  Create a new angular project
+-  Create a component
+-  Add task
+-  Mark task as done
+-  Delete a task
+-  Save to local storage
+-  Summary
```

Comment:
There is a way to link headings. For example,

- [Prerequisites](#prerequisites) => [Prerequisites] then concatenate with (#prerequisites).Check markdown docs if you're stuck!